oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -416,6 +416,9 @@ Flask:
 Requests:
   id: 7002
   languages: Python
+PyMysql:
```

Comment:
I think this is MySQL driver in Python, we need a mapping for this to show MySQL server icon in the topology, 

Ref to, https://github.com/apache/skywalking-rocketbot-ui/pull/331#issuecomment-660499481

Reply:
OK